### ISSUE TITLE
crash building ubuntu ami

### ISSUE THREAD

**Issue by schmidlop** [hashicorp/packer#6302]
Worked correctly on subsequent execution.
host os is mac 10.13.4

Contents of crash.log:
```
2018/05/24 12:35:08 [INFO] Packer version: 1.1.0
2018/05/24 12:35:08 Packer Target OS/Arch: darwin amd64
2018/05/24 12:35:08 Built with Go Version: go1.9
2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 Using internal plugin for amazon-instance
2018/05/24 12:35:08 Using internal plugin for file
2018/05/24 12:35:08 Using internal plugin for hyperv-iso
2018/05/24 12:35:08 Using internal plugin for qemu
2018/05/24 12:35:08 Using internal plugin for amazon-ebsvolume
2018/05/24 12:35:08 Using internal plugin for cloudstack
2018/05/24 12:35:08 Using internal plugin for virtualbox-ovf
2018/05/24 12:35:08 Using internal plugin for amazon-ebs
2018/05/24 12:35:08 Using internal plugin for azure-arm
2018/05/24 12:35:08 Using internal plugin for docker
2018/05/24 12:35:08 Using internal plugin for lxc
2018/05/24 12:35:08 Using internal plugin for parallels-iso
2018/05/24 12:35:08 Using internal plugin for null
2018/05/24 12:35:08 Using internal plugin for vmware-vmx
2018/05/24 12:35:08 Using internal plugin for oracle-oci
2018/05/24 12:35:08 Using internal plugin for alicloud-ecs
2018/05/24 12:35:08 Using internal plugin for amazon-chroot
2018/05/24 12:35:08 Using internal plugin for amazon-ebssurrogate
2018/05/24 12:35:08 Using internal plugin for googlecompute
2018/05/24 12:35:08 Using internal plugin for lxd
2018/05/24 12:35:08 Using internal plugin for parallels-pvm
2018/05/24 12:35:08 Using internal plugin for profitbricks
2018/05/24 12:35:08 Using internal plugin for virtualbox-iso
2018/05/24 12:35:08 Using internal plugin for vmware-iso
2018/05/24 12:35:08 Using internal plugin for digitalocean
2018/05/24 12:35:08 Using internal plugin for openstack
2018/05/24 12:35:08 Using internal plugin for oneandone
2018/05/24 12:35:08 Using internal plugin for triton
2018/05/24 12:35:08 Using internal plugin for converge
2018/05/24 12:35:08 Using internal plugin for shell-local
2018/05/24 12:35:08 Using internal plugin for windows-restart
2018/05/24 12:35:08 Using internal plugin for windows-shell
2018/05/24 12:35:08 Using internal plugin for salt-masterless
2018/05/24 12:35:08 Using internal plugin for shell
2018/05/24 12:35:08 Using internal plugin for ansible
2018/05/24 12:35:08 Using internal plugin for chef-solo
2018/05/24 12:35:08 Using internal plugin for file
2018/05/24 12:35:08 Using internal plugin for puppet-masterless
2018/05/24 12:35:08 Using internal plugin for puppet-server
2018/05/24 12:35:08 Using internal plugin for ansible-local
2018/05/24 12:35:08 Using internal plugin for chef-client
2018/05/24 12:35:08 Using internal plugin for powershell
2018/05/24 12:35:08 Using internal plugin for shell-local
2018/05/24 12:35:08 Using internal plugin for vsphere-template
2018/05/24 12:35:08 Using internal plugin for artifice
2018/05/24 12:35:08 Using internal plugin for checksum
2018/05/24 12:35:08 Using internal plugin for docker-tag
2018/05/24 12:35:08 Using internal plugin for atlas
2018/05/24 12:35:08 Using internal plugin for docker-import
2018/05/24 12:35:08 Using internal plugin for manifest
2018/05/24 12:35:08 Using internal plugin for vagrant
2018/05/24 12:35:08 Using internal plugin for vsphere
2018/05/24 12:35:08 Using internal plugin for docker-push
2018/05/24 12:35:08 Using internal plugin for docker-save
2018/05/24 12:35:08 Using internal plugin for googlecompute-export
2018/05/24 12:35:08 Using internal plugin for vagrant-cloud
2018/05/24 12:35:08 Using internal plugin for alicloud-import
2018/05/24 12:35:08 Using internal plugin for amazon-import
2018/05/24 12:35:08 Using internal plugin for compress
2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 Attempting to open config file: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 [WARN] Config file doesn't exist: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[azure-arm:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-azure-arm lxd:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-lxd parallels-pvm:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-parallels-pvm oneandone:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-oneandone amazon-ebs:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-ebsvmware-vmx:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-vmware-vmx openstack:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-openstack profitbricks:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-profitbricks virtualbox-iso:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-virtualbox-iso amazon-instance:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-instance amazon-ebsvolume:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-ebsvolume lxc:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-lxc null:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-null amazon-chroot:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-chroot digitalocean:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-digitalocean triton:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-triton cloudstack:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-cloudstack docker:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-docker parallels-iso:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-parallels-iso qemu:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-qemu amazon-ebssurrogate:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-amazon-ebssurrogate vmware-iso:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-vmware-iso googlecompute:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-googlecomputefile:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-file oracle-oci:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-oracle-oci alicloud-ecs:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-alicloud-ecs virtualbox-ovf:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-virtualbox-ovf hyperv-iso:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-builder-hyperv-iso] PostProcessors:map[vsphere-template:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vsphere-template docker-tag:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-tag vagrant:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vagrant docker-save:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-save shell-local:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-shell-local docker-import:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-import googlecompute-export:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-googlecompute-export alicloud-import:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-alicloud-import amazon-import:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-amazon-import compress:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-compress artifice:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-artifice checksum:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-checksum atlas:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-atlas manifest:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-manifest vsphere:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vsphere docker-push:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-docker-push vagrant-cloud:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-post-processor-vagrant-cloud] Provisioners:map[shell:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-shell ansible:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-ansible ansible-local:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-ansible-local puppet-masterless:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-puppet-masterless powershell:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-powershell converge:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-converge windows-restart:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-windows-restart windows-shell:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-windows-shell chef-solo:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-chef-solo shell-local:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-shell-local salt-masterless:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-salt-masterless file:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-file puppet-server:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-puppet-server chef-client:/Users/pbb_bschmitt/bin/packer-PACKERSPACE-plugin-PACKERSPACE-packer-provisioner-chef-client]}
2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 Setting cache directory: /Users/pbb_bschmitt/Documents/repos/phoenix/packer_cache
2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 Loading builder: amazon-ebs
2018/05/24 12:35:08 Plugin could not be found. Checking same directory as executable.
2018/05/24 12:35:08 Current exe path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Creating plugin client for path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Starting plugin: /Users/pbb_bschmitt/bin/packer []string{"/Users/pbb_bschmitt/bin/packer", "plugin", "packer-builder-amazon-ebs"}
2018/05/24 12:35:08 Waiting for RPC address for: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [INFO] Packer version: 1.1.0
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer Target OS/Arch: darwin amd64
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Built with Go Version: go1.9
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Attempting to open config file: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [WARN] Config file doesn't exist: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Setting cache directory: /Users/pbb_bschmitt/Documents/repos/phoenix/packer_cache
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 args: []string{"packer-builder-amazon-ebs"}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin minimum port: 10000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin maximum port: 25000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin address: unix /var/folders/j1/s7cqcwjx7qjg3cxt_b32g1gr0000gn/T/packer-plugin084970124
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Waiting for connection...
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Serving a plugin connection...
2018/05/24 12:35:08 Loading provisioner: shell
2018/05/24 12:35:08 Plugin could not be found. Checking same directory as executable.
2018/05/24 12:35:08 Current exe path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Creating plugin client for path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Starting plugin: /Users/pbb_bschmitt/bin/packer []string{"/Users/pbb_bschmitt/bin/packer", "plugin", "packer-provisioner-shell"}
2018/05/24 12:35:08 Waiting for RPC address for: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [INFO] Packer version: 1.1.0
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer Target OS/Arch: darwin amd64
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Built with Go Version: go1.9
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Attempting to open config file: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [WARN] Config file doesn't exist: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Setting cache directory: /Users/pbb_bschmitt/Documents/repos/phoenix/packer_cache
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 args: []string{"packer-provisioner-shell"}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin minimum port: 10000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin maximum port: 25000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin address: unix /var/folders/j1/s7cqcwjx7qjg3cxt_b32g1gr0000gn/T/packer-plugin999863136
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Waiting for connection...
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Serving a plugin connection...
2018/05/24 12:35:08 Loading provisioner: file
2018/05/24 12:35:08 Plugin could not be found. Checking same directory as executable.
2018/05/24 12:35:08 Current exe path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Creating plugin client for path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Starting plugin: /Users/pbb_bschmitt/bin/packer []string{"/Users/pbb_bschmitt/bin/packer", "plugin", "packer-provisioner-file"}
2018/05/24 12:35:08 Waiting for RPC address for: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [INFO] Packer version: 1.1.0
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer Target OS/Arch: darwin amd64
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Built with Go Version: go1.9
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Attempting to open config file: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [WARN] Config file doesn't exist: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Setting cache directory: /Users/pbb_bschmitt/Documents/repos/phoenix/packer_cache
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 args: []string{"packer-provisioner-file"}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin minimum port: 10000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin maximum port: 25000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin address: unix /var/folders/j1/s7cqcwjx7qjg3cxt_b32g1gr0000gn/T/packer-plugin815993724
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Waiting for connection...
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Serving a plugin connection...
2018/05/24 12:35:08 Loading provisioner: shell
2018/05/24 12:35:08 Plugin could not be found. Checking same directory as executable.
2018/05/24 12:35:08 Current exe path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Creating plugin client for path: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 Starting plugin: /Users/pbb_bschmitt/bin/packer []string{"/Users/pbb_bschmitt/bin/packer", "plugin", "packer-provisioner-shell"}
2018/05/24 12:35:08 Waiting for RPC address for: /Users/pbb_bschmitt/bin/packer
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [INFO] Packer version: 1.1.0
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer Target OS/Arch: darwin amd64
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Built with Go Version: go1.9
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Attempting to open config file: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 [WARN] Config file doesn't exist: /Users/pbb_bschmitt/.packerconfig
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[] PostProcessors:map[] Provisioners:map[]}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Setting cache directory: /Users/pbb_bschmitt/Documents/repos/phoenix/packer_cache
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 args: []string{"packer-provisioner-shell"}
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin minimum port: 10000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin maximum port: 25000
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Detected home directory from env var: /Users/pbb_bschmitt
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Plugin address: unix /var/folders/j1/s7cqcwjx7qjg3cxt_b32g1gr0000gn/T/packer-plugin263480136
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Waiting for connection...
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Serving a plugin connection...
2018/05/24 12:35:08 ui: amazon-ebs output will be in this color.
2018/05/24 12:35:08 ui:
2018/05/24 12:35:08 Build debug mode: false
2018/05/24 12:35:08 Force build: false
2018/05/24 12:35:08 On error:
2018/05/24 12:35:08 Preparing build: amazon-ebs
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Config: {PackerConfig:{PackerBuildName:amazon-ebs PackerBuilderType:amazon-ebs PackerDebug:false PackerForce:false PackerOnError: PackerUserVars:map[aws_access_key: aws_secret_key: docker_password:]} AccessConfig:{AccessKey: CustomEndpointEc2: MFACode: ProfileName: RawRegion:us-east-1 SecretKey: SkipValidation:false Token: session:<nil>} AMIConfig:{AMIName:pbb-docker AMIDescription: AMIVirtType: AMIUsers:[] AMIGroups:[] AMIProductCodes:[] AMIRegions:[] AMISkipRegionValidation:false AMITags:map[] AMIENASupport:false AMISriovNetSupport:false AMIForceDeregister:true AMIForceDeleteSnapshot:false AMIEncryptBootVolume:false AMIKmsKeyId: AMIRegionKMSKeyIDs:map[] SnapshotTags:map[] SnapshotUsers:[] SnapshotGroups:[]} BlockDevices:{AMIBlockDevices:{AMIMappings:[]} LaunchBlockDevices:{LaunchMappings:[]}} RunConfig:{AssociatePublicIpAddress:false AvailabilityZone: EbsOptimized:false IamInstanceProfile: InstanceType:t2.micro RunTags:map[] SourceAmi: SourceAmiFilter:{Filters:map[0xc420475c20:0xc420475c40 0xc420475c60:0xc420475c80 0xc420475ca0:0xc420475cc0] Owners:[0xc420475d40] MostRecent:true} SpotPrice: SpotPriceAutoProduct: DisableStopInstance:false SecurityGroupId: SecurityGroupIds:[] SubnetId: TemporaryKeyPairName:packer_5b06e9bc-2af1-1fda-f594-18f1662ad936 UserData: UserDataFile: WindowsPasswordTimeout:20m0s VpcId: InstanceInitiatedShutdownBehavior:stop Comm:{Type:ssh SSHHost: SSHPort:22 SSHUsername:ubuntu SSHPassword: SSHPrivateKey: SSHPty:false SSHTimeout:5m0s SSHAgentAuth:false SSHDisableAgentForwarding:false SSHHandshakeAttempts:10 SSHBastionHost: SSHBastionPort:0 SSHBastionAgentAuth:false SSHBastionUsername: SSHBastionPassword: SSHBastionPrivateKey: SSHFileTransferMethod:scp WinRMUser: WinRMPassword: WinRMHost: WinRMPort:0 WinRMTimeout:0s WinRMUseSSL:false WinRMInsecure:false WinRMUseNTLM:false WinRMTransportDecorator:<nil>} SSHKeyPairName: SSHPrivateIp:false} VolumeRunTags:map[] ctx:{Data:<nil> Funcs:map[clean_ami_name:0x18603f0] UserVariables:map[docker_password: aws_access_key: aws_secret_key:] EnableEnv:false BuildName:amazon-ebs BuildType:amazon-ebs TemplatePath:/Users/pbb_bschmitt/Documents/repos/phoenix/amis/pbb-docker.json}}
2018/05/24 12:35:08 Waiting on builds to complete...
2018/05/24 12:35:08 Starting build run: amazon-ebs
2018/05/24 12:35:08 Running builder: amazon-ebs
2018/05/24 12:35:08 [INFO] (telemetry) Starting builder amazon-ebs
2018/05/24 12:35:08 ui: ==> amazon-ebs: Force Deregister flag found, skipping prevalidating AMI Name
2018/05/24 12:35:08 packer: 2018/05/24 12:35:08 Using AMI Filters {
2018/05/24 12:35:08 packer:   Filters: [{
2018/05/24 12:35:08 packer:       Name: "name",
2018/05/24 12:35:08 packer:       Values: ["ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*"]
2018/05/24 12:35:08 packer:     },{
2018/05/24 12:35:08 packer:       Name: "root-device-type",
2018/05/24 12:35:08 packer:       Values: ["ebs"]
2018/05/24 12:35:08 packer:     },{
2018/05/24 12:35:08 packer:       Name: "virtualization-type",
2018/05/24 12:35:08 packer:       Values: ["hvm"]
2018/05/24 12:35:08 packer:     }],
2018/05/24 12:35:08 packer:   Owners: ["099720109477"]
2018/05/24 12:35:08 packer: }
2018/05/24 12:35:10 ui:     amazon-ebs: Found Image ID: ami-a4dc46db
2018/05/24 12:35:10 ui: ==> amazon-ebs: Creating temporary keypair: packer_5b06e9bc-2af1-1fda-f594-18f1662ad936
2018/05/24 12:35:10 ui: ==> amazon-ebs: Creating temporary security group for this instance: packer_5b06e9be-927e-4d2b-63d0-09a5fca933d8
2018/05/24 12:35:10 ui: ==> amazon-ebs: Authorizing access to port 22 on the temporary security group...
2018/05/24 12:35:10 packer: 2018/05/24 12:35:10 [DEBUG] Waiting for temporary security group: sg-072d594f
2018/05/24 12:35:10 ui: ==> amazon-ebs: Deleting temporary keypair...
2018/05/24 12:35:11 packer: panic: runtime error: invalid memory address or nil pointer dereference
2018/05/24 12:35:11 packer: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16ddced]
2018/05/24 12:35:11 packer:
2018/05/24 12:35:11 packer: goroutine 119 [running]:
2018/05/24 12:35:11 packer: github.com/hashicorp/packer/vendor/github.com/aws/aws-sdk-go/aws/request.Waiter.WaitWithContext(0x2c68f2c, 0x16, 0xc4201123f0, 0x3, 0x3, 0x3cc5380, 0xc420166020, 0x28, 0x0, 0x0, ...)
2018/05/24 12:35:11 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/vendor/github.com/aws/aws-sdk-go/aws/request/waiter.go:195 +0x18d
2018/05/24 12:35:11 packer: github.com/hashicorp/packer/builder/amazon/common.waitUntilSecurityGroupExists(0xc420166098, 0xc4204e8190, 0xc4202e36e8, 0x1)
2018/05/24 12:35:11 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/builder/amazon/common/step_security_group.go:193 +0x262
2018/05/24 12:35:11 packer: github.com/hashicorp/packer/builder/amazon/common.(*StepSecurityGroup).Run(0xc420063b40, 0x3cd8ac0, 0xc4202437a0, 0x0)
2018/05/24 12:35:11 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/builder/amazon/common/step_security_group.go:108 +0x9d7
2018/05/24 12:35:11 packer: github.com/hashicorp/packer/vendor/github.com/mitchellh/multistep.(*BasicRunner).Run(0xc420063c00, 0x3cd8ac0, 0xc4202437a0)
2018/05/24 12:35:11 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/vendor/github.com/mitchellh/multistep/basic_runner.go:70 +0x1ee
2018/05/24 12:35:11 packer: github.com/hashicorp/packer/builder/amazon/ebs.(*Builder).Run(0xc420013900, 0x3cdbb60, 0xc4200e0340, 0x3cd2cc0, 0xc420048b50, 0x3cda4c0, 0xc420166018, 0x80000000000, 0x28, 0x28, ...)
2018/05/24 12:35:11 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/builder/amazon/ebs/builder.go:225 +0x15ca
2018/05/24 12:35:11 packer: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Run(0xc4200e1220, 0x1, 0xc420018220, 0x0, 0x0)
2018/05/24 12:35:11 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/packer/rpc/builder.go:93 +0x1dd
2018/05/24 12:35:11 packer: reflect.Value.call(0xc4200688a0, 0xc42000e588, 0x13, 0x2c4486b, 0x4, 0xc42041ff20, 0x3, 0x3, 0xc420434000, 0xc420430000, ...)
2018/05/24 12:35:11 packer:     /usr/local/go/src/reflect/value.go:434 +0x906
2018/05/24 12:35:11 packer: reflect.Value.Call(0xc4200688a0, 0xc42000e588, 0x13, 0xc420414f20, 0x3, 0x3, 0xc420414f48, 0xc4200228a0, 0x0)
2018/05/24 12:35:11 packer:     /usr/local/go/src/reflect/value.go:302 +0xa4
2018/05/24 12:35:11 packer: net/rpc.(*service).call(0xc420063ec0, 0xc420086910, 0xc420019bc0, 0xc4200faa00, 0xc4200e1520, 0x26c2960, 0xc4200181fc, 0x18a, 0x2669b20, 0xc420018220, ...)
2018/05/24 12:35:11 packer:     /usr/local/go/src/net/rpc/server.go:381 +0x142
2018/05/24 12:35:11 packer: created by net/rpc.(*Server).ServeCodec
2018/05/24 12:35:11 packer:     /usr/local/go/src/net/rpc/server.go:475 +0x36b
2018/05/24 12:35:11 [INFO] (telemetry) ending amazon-ebs
2018/05/24 12:35:11 [INFO] (telemetry) found error: unexpected EOF
2018/05/24 12:35:11 ui error: Build 'amazon-ebs' errored: unexpected EOF
2018/05/24 12:35:11 Builds completed. Waiting on interrupt barrier...
2018/05/24 12:35:11 machine readable: error-count []string{"1"}
2018/05/24 12:35:11 ui error:
==> Some builds didn't complete successfully and had errors:
2018/05/24 12:35:11 machine readable: amazon-ebs,error []string{"unexpected EOF"}
2018/05/24 12:35:11 ui error: --> amazon-ebs: unexpected EOF
2018/05/24 12:35:11 ui:
==> Builds finished but no artifacts were created.
2018/05/24 12:35:11 /Users/pbb_bschmitt/bin/packer: plugin process exited
2018/05/24 12:35:11 waiting for all plugin processes to complete...
2018/05/24 12:35:11 /Users/pbb_bschmitt/bin/packer: plugin process exited
2018/05/24 12:35:11 /Users/pbb_bschmitt/bin/packer: plugin process exited
2018/05/24 12:35:11 /Users/pbb_bschmitt/bin/packer: plugin process exited
```

**Comment by rickard-von-essen** [comment]
Can you reproduce this with version 1.2.3?

**Comment by mwhooker** [comment]
Thanks for the report!

going to mark this as a dupe of #5535. It should be fixed in later versions. If you update and still get this, please open a new issue with the new crash log and we'll get it taken care of.
